etc/docs.yaml.dist

```diff
@@ -0,0 +1,15 @@
+# This is an example configuration file for displaying documentation links in
+# the team interface.
+
+# This file should be an array with items to show.
+
+  # Add a link to a file under webapp/public/docs/
+- name: Team guide
+  file: team.pdf
```

Comment:
This distinction between file and link can be removed. If you'd specify link: team.pdf it would work the same but as a relative url.

Reply:
But not really, since the link is not /team/team.pdf but /docs/team.pdf. I can’t use a link starting with / since it might run under a subdir. Thoughts?

Reply:
Not if you make the route be /docs/ instead of /team/docs I think? So if you place a `<a href="team.pdf">` there it just works. And relative links are no longer special.

Reply:
If you don't want to change the route, add ../ before any relative link.

Reply:
This will work yes. I will change it. I don’t like changing the url because we then need to move the action to its own controller, but ../ seems fine. Is there then any reason to put the docs folder in git? You can put pdfs anywhere. Although in maintainer mode this way you don’t have files git will see as unadded